tools/test-js-with-node

```diff
@@ -39,7 +39,7 @@ enforce_fully_covered = {
     'static/js/compose_ui.js',
     # Temporarily removed until we finish merging emoji commits.
     # 'static/js/composebox_typeahead.js',
-    'static/js/dict.js',
+    'static/js/dict.ts',
```

Comment:
This will be addressed in https://github.com/zulip/zulip/issues/12000, Istanbul currently only recognises files with `.js` extension.